internlm/train/pipeline.py

```diff
@@ -299,7 +299,7 @@ def initialize_optimizer(model: Union[nn.Module, nn.ModuleList], isp_communicato
     adam_extra_kwargs = {}
     # set fused=True to avoid nan grad norm when model size is larger and use_fp32_norm=True
 
-    if internlm_accelerator.get_accelerator_backend() == AcceleratorType.NPU:
+    if get_accelerator().get_accelerator_backend() == AcceleratorType.NPU:
```

Comment:
internlm_accelerator = get_accelerator() has already been defined, so no need to use get_accelerator()?